lib/Test/PostgreSQL.pm

```diff
@@ -116,6 +123,13 @@ has extra_initdb_args => (
   default => "",
 );
 
+has use_socket =>
+    (
+     is  => "ro",
+     isa     => Bool,
+     default => 0,
+    );
```

Comment:
Can you make this in a consistent style with the other attributes? (move the parens)

Reply:
Sure; which indentation style are you using?

Reply:
As in, what settings should I give perltidy?